azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -175,7 +150,16 @@ public Props getProps() {
   }
 
   public List<Project> getProjects() {
-    return new ArrayList<Project>(projectsById.values());
+    List<Project> projects;
```

Comment:
Intellij will give some optimization suggestions like the initialization is redundant and should be removed.